build.md

```diff
@@ -1 +1,7 @@
-# Building and running Piconot
\ No newline at end of file
+# Building and running Piconot
+
+Simply run
+
+```
+$ sbt "project piconot-external" "run path/to/maze.txt path/to/program.bot"
```

Comment:
When you write "simply run" I expect to be able to copy and paste the command. You should write "To run, write the command in the following format." It'd be nice to include a sample of commands that I could run.